### ISSUE TITLE
Clarify how block vs inline math should be handled

### ISSUE THREAD

**Issue by AndersEkl** [nlbdev/mathml-guidelines#24]
The `<math>` element can be both a block or an inline element. The `display` attribute doesn't seem to change the actual property of the `<math>` element, regardless of its value, but merely affects the way the math is displayed.

Standard HTML validation allows `<math display="block">` in inline contexts, even though a browser effectively handles and displays it as a block element. Some other applications, however, treat it like an actual block element. Webarch, for instance.

My suggestion is to regard `<math display="block">` as a block element.

**Comment by kvile-com** [comment]
If we have:

"Did you know that
1+1=2
already?"

it should not be marked up as:
```
<p>Did you know that</p>
<p><math><...>1+1=2</...></math></p>
<p>already?</p>
```

because it might generate unwanted pauses. It should be read the same way as `<p>Did you know that <math display="block"><...>1+1=2</...></math> already?</p>`

**Comment by samimaattaCelia** [comment]
Here is also reading on how MathML should be marked up together with HTML: [MathML (HTML spec)](https://html.spec.whatwg.org/multipage/embedded-content-other.html#mathml). The text is a bit dense for me, but the mark up example is like what @kvile-com wrote.

The MathML code is in inside the `<p>` element. This also makes it possible for proper sentence structure and grammar to be used.

```html
<h1>The quadratic formula</h1>
  <p>
   <math>
    <mi>x</mi>
    <mo>=</mo>
    <mfrac>
     <mrow>
      <mo form="prefix">−</mo> <mi>b</mi>
      <mo>±</mo>
      <msqrt>
       <msup> <mi>b</mi> <mn>2</mn> </msup>
       <mo>−</mo>
       <mn>4</mn> <mo>⁢</mo> <mi>a</mi> <mo>⁢</mo> <mi>c</mi>
      </msqrt>
     </mrow>
     <mrow>
      <mn>2</mn> <mo>⁢</mo> <mi>a</mi>
     </mrow>
    </mfrac>
   </math>
  </p>
```

**Comment by samimaattaCelia** [comment]
Yes, I think it would mean that, since math tends to always be part of a sentence or a paragraph. I'm not sure if this is the intention of MathML and it could be clarified from Neil.

Maybe there are exceptions, but I think it would be complicated and I'm not sure which would be more common.

**Comment by TimArborealis** [comment]
The Block vs inline section needs to be fleshed out to include this. Or does the information belong somewhere else?

**Comment by AndersEkl** [comment]
@TimArborealis I am on it! :)

**Comment by TimArborealis** [comment]
Clarify the block equation example so it isn't at risk of seeming contradictory.
